ts_src/psetv2/blinder.ts

```diff
@@ -0,0 +1,622 @@
+import { ZERO } from '../transaction';
+import { Pset } from './pset';
+
+export interface IssuanceBlindingArgs {
+  index: number;
+  issuanceAsset: Buffer;
+  issuanceToken: Buffer;
+  issuanceValueCommitment: Buffer;
+  issuanceTokenCommitment: Buffer;
+  issuanceValueRangeProof: Buffer;
+  issuanceTokenRangeProof: Buffer;
+  issuanceValueBlindProof: Buffer;
+  issuanceTokenBlindProof: Buffer;
+  issuanceValueBlinder: Buffer;
+  issuanceTokenBlinder: Buffer;
+}
+
+export interface OutputBlindingArgs {
+  index: number;
+  nonce: Buffer;
+  nonceCommitment: Buffer;
+  valueCommitment: Buffer;
+  assetCommitment: Buffer;
+  valueRangeProof: Buffer;
+  assetSurjectionProof: Buffer;
+  valueBlindProof: Buffer;
+  assetBlindProof: Buffer;
+  valueBlinder: Buffer;
+  assetBlinder: Buffer;
+}
+
+export interface OwnedInput {
```

Comment:
We could replace by `type OwnedInput = { index: number } & UnblindedOutput['unblindData']`. The idea is to be able to "cast" utxo to OwnedInput.